examples/mqtt/samplePropMqtt.conf

```diff
@@ -0,0 +1,36 @@
+#### [DEFAULT] ####
+
+EdgeSystemName = "EdgeSystem-Name"
+DeviceName = "Device-Name"
+MetricName = "Metric-Name"
+MetricWithOwnTsName = "Metric-With-Own-Ts-Name"
+BulkCollectedMetricName = "Bulk-Collected-Metric-Name"
+
+SystemPropList = {"Country":"USA-G", "State":"California", "City":"Palo Alto", "Location":"VMware HQ", "Building":"Promontory H Lab", "Floor":"First Floor"}
+DevicePropList = {"Country":"USA-G", "State":"California", "City":"Palo Alto", "Location":"VMware HQ", "Building":"Promontory H Lab", "Floor":"First Floor"}
+
+#### [IOTCC] ####
+
+WebSocketUrl = "Websocket-address-url"
+IotCCUID = "Username"
+IotCCPassword = "Password"
+
+#### [MQTT] ####
+
+MqttChannel = "mqtt-channel"
+MqttSubChannel1 = "mqtt-subchannel1"
+MqttSubChannel2 = "mqtt-subchannel2"
+BrokerIP = "mqtt-broker"
+BrokerPort = "mqtt-broker-port"
+keepalive = 60
+cacert = "/etc/broker-name/ca_certificates/ca.crt"
```

Comment:
Should these be set in /etc/liota/conf/ or /etc/liota/mqtt/conf/ instead of /etc/ directly?